### ISSUE TITLE
html5 validation

### ISSUE THREAD

**Issue by danteand** [adrian3/wp-google-fonts#2]
Does the link in the header generated by the plugin validate for html5?

A link of this form seems to generate an error:

http://fonts.googleapis.com/css?family=Open+Sans:300,400|Montserrat:400|Open+Sans+Condensed:300&subset=latin

but encoded like this, it seems to validate

http://fonts.googleapis.com/css?family=Open%20Sans:300,400%7CMontserrat:400%7COpen%20Sans%20Condensed:300&amp;subset=latin

both seem to work for google

**Comment by rcemory** [comment]
I agree that the ampersand should be escaped in order for html5 validation. 

In addition, the ending tag for the link should end in a single caret > instead of the />. Minor bug that does not affect the operation of this plugin but should be corrected in the next release.
